deps/chakrashim/src/v8object.cc

```diff
@@ -66,16 +66,11 @@ bool Object::Set(Handle<Value> key, Handle<Value> value) {
 
 Maybe<bool> Object::Set(Handle<Value> key, Handle<Value> value,
                         PropertyAttribute attribs, bool force) {
-  JsPropertyIdRef idRef;
-
-  if (jsrt::GetPropertyIdFromValue((JsValueRef)*key, &idRef) != JsNoError) {
```

Comment:
Previously this code path worked for Symbols as well as strings; do you know if that is ever used?

Reply:
Actually, it also tried to work for all values, and would stringify things that weren't strings or symbols.

Reply:
consider symbol is there. I'm not sure if we `should!` support other types though.